lib/proxy_logic_13/proxy_logic_13.js

```diff
@@ -6158,7 +6158,8 @@ function evtConversationDialing(data) {
       typeof data.chSource !== 'string' &&
       typeof data.callerNum !== 'string' &&
       typeof data.chDestExten !== 'string' &&
-      typeof data.chSourceExten !== 'string') {
+      typeof data.chSourceExten !== 'string' &&
+      typeof data.destUniqueId !== 'string') {
```

Comment:
Revert this change. If `destUniqueId` is not present,  the `EVT_EXTEN_DIALING` event will never be fired.
The asterisk doc says the `destUniqueId` should have a vale, but we can't put so much trust on it.
```suggestion
      typeof data.chSourceExten !== 'string') {
```